Tests/HDF5Benchmark/GNUmakefile

```diff
@@ -23,17 +23,19 @@ USE_PARTICLES = TRUE
 EBASE     = main
 
 USE_HDF5  = TRUE
-# HDF5_HOME = /gpfs/alpine/csc300/world-shared/gnu_build/hdf5/develop_build/hdf5
 HDF5_HOME = $(OLCF_HDF5_ROOT)
+# HDF5_HOME = /lustre/orion/geo130/world-shared/gnu/hdf5.1.14.1.sync.cmake
+# HDF5_HOME = $(HDF5_ROOT)
+# HDF5_HOME = /lustre/orion/geo130/world-shared/gnu/hdf5.1.14.1.async.cmake
 
-# To use HDF5 with ZFP compression, follow the instructions at https://h5z-zfp.readthedocs.io to compile ZFP and H5Z_ZFP plugin, or use the following gcc build on Summit
+# To use HDF5 with ZFP compression, follow the instructions at https://h5z-zfp.readthedocs.io to compile ZFP and H5Z_ZFP plugin, or use the following gcc build on Frontier
 USE_HDF5_ZFP = FALSE
-H5Z_HOME  = /gpfs/alpine/csc300/world-shared/gnu_build/H5Z-ZFP/install
-ZFP_HOME  = /gpfs/alpine/csc300/world-shared/gnu_build/zfp
+H5Z_HOME  = /lustre/orion/geo130/world-shared/gnu/H5Z-ZFP
+ZFP_HOME  = /lustre/orion/geo130/world-shared/gnu/zfp
```

Comment:
Does it makes sense to leave the Summit one in, but commented out? 

Also, are these available for everyone to use, or do you need access to `geo130`?

Reply:
Just add the Summit ones that are commented out.
Everyone should be able to access the "world-shared" locations on both Summit and Frontier.